ui/js/controllers/jobs.js

```diff
@@ -106,28 +106,6 @@ treeherderApp.controller('ResultSetCtrl', [
         $scope.getCountText = function(resultStatus) {
             return thResultStatusInfo(resultStatus).countText;
         };
-        $scope.viewJob = function(job) {
```

Comment:
Looks like this function is still referenced here: https://github.com/mozilla/treeherder/blob/cd99209f8d1d218e3d143ea4a6bb97b2209e2151/ui/js/controllers/jobs.js#L143-L143

iirc, that was so that if you pin all the shown jobs, it'll select the first one so that the pinboard would be shown.  Or at least give some indication that the action succeeded.